cpp/tests/io/json/json_test.cpp

```diff
@@ -16,84 +16,284 @@
 
 #include <cudf.h>
 
-#include<cuda_runtime.h>
+#include <cuda_runtime.h>
 
-#include <gtest/gtest.h>
 #include <gmock/gmock.h>
+#include <gtest/gtest.h>
 
 #include <sys/stat.h>
 
-#include <iostream>
 #include <fstream>
-#include <vector>
+#include <iostream>
 #include <string>
+#include <vector>
 
 #include <cstdlib>
 
+#include <nvstrings/NVStrings.h>
+
 #include "io/utilities/parsing_utils.cuh"
 
-using std::vector;
 using std::string;
+using std::vector;
+
+bool checkFile(std::string fname) {
+  struct stat st;
+  return (stat(fname.c_str(), &st) ? 0 : 1);
+}
+
+template <typename T> std::vector<T> gdf_column_to_host(gdf_column *const col) {
+  auto m_hostdata = std::vector<T>(col->size);
+  cudaMemcpy(m_hostdata.data(), col->data, sizeof(T) * col->size, cudaMemcpyDeviceToHost);
+  return m_hostdata;
+}
+
+TEST(gdf_json_test, SquareBrackets) {
```

Comment:
Include `tests/utilities/cudf_test_fixtures.h` and have test cases use a fixture derived from `GdfTest` (like `dlpack` and `decomp` gtests, to ensure RMM is always initialized.

Reply:
Done. Please check if this is what you had in mind.

Reply:
I think it needs to be `TEST_F` instead of `TEST`?

Reply:
Yep. Fixed.